src/openforms/js/components/admin/forms/ReactSelect.js

```diff
@@ -31,19 +31,43 @@ const styles = {
   }),
 };
 
+/**
+ * A select dropdown backed by react-select for regular usage.
+ *
+ * Any additional props are forwarded to the underlying ReactSelect component.
+ */
```

Comment:
```suggestion
/**
 * A select dropdown backed by react-select for legacy usage.
 *
 * Any additional props are forwarded to the underlying ReactSelect component.
 *
 * @deprecated - if possible, refactor the form to use Formik and use the Formik-enabled
 * variant.
 */
```

Our ambition is to manage all form state via Formik - not a reality yet, but let's make clear that this is an escape hatch :)

Reply:
Ah yes, good one! I'll add it 👍